src/mono/mono/mini/interp/transform.c

```diff
@@ -4517,6 +4515,8 @@ interp_method_compute_offsets (TransformData *td, InterpMethod *imethod, MonoMet
 
 #ifdef MONO_ARCH_HAVE_SWIFTCALL
 	if (swift_error_index < 0 && mono_method_signature_has_ext_callconv (sig, MONO_EXT_CALLCONV_SWIFTCALL)) {
+		MonoClass *swift_error = mono_class_try_get_swift_error_class ();
+		MonoClass *swift_error_ptr = mono_class_create_ptr (m_class_get_this_arg (swift_error));
```

Comment:
I think we should still do a null check here in case there are CallConvSwift calls without any usages of SwiftError.

Reply:
Note, we have the same pattern in `marshal.c`, and `mini-<arch>.c` files. 

However, if I understand it correctly, error here would mean that `CallConvSwift` is set and `System.Runtime.InteropServices.Swift` got trimmed out. Sounds unlikely but perhaps possible...

Reply:
Thanks, added. I don't think it should report any error message since it will just skip the `swift_error` path.